source/game/system/map/MapdataCannonPoint.hh

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "game/system/map/MapdataAccessorBase.hh"
+
+#include <egg/math/Vector.hh>
+#include <game/kart/KartMove.hh>
+
+namespace System {
+class MapdataCannonPoint {
```

Comment:
Really pedantic, but all our other files have a newline between the namespace and class.
```suggestion

class MapdataCannonPoint {
```